scripts/sql/xavier-changes-2016_April/aatamsDB_queries_#268.sql

```diff
@@ -0,0 +1,70 @@
+﻿set search_path = aatams, public;
+
+begin;
+
+-- 1. Clean up animal_release table by deleting animal_release records with no corresponding surgery
+DELETE FROM animal_release ar
+WHERE NOT EXISTS (SELECT release_id FROM surgery s WHERE s.release_id = ar.id);
```

Comment:
Do you have the urls for the list of backups? My understanding was that they are not staggered in time - meaning there would only be a 2-week window (or whatever the backup period is set to) to correct a mistake.